locales/LocalesResources.d.ts

```diff
@@ -0,0 +1,31 @@
+type LocalesResources = {
+  '404': typeof import('../pages/404.de').default
+  qr: typeof import('../pages/qr.de').default
+  index: typeof import('../pages/index.de').default
+  _error: typeof import('../pages/_error.de').default
+  corona: typeof import('../pages/corona.de').default
+  checkin: typeof import('../pages/checkin.de').default
+  'my-checkins': typeof import('../pages/my-checkins.de').default
+  'business/index': typeof import('../pages/business/index.de').default
+  'business/login': typeof import('../pages/business/login.de').default
+  'business/logout': typeof import('../pages/business/logout.de').default
+  'business/profile': typeof import('../pages/business/profile.de').default
+  'business/dashboard': typeof import('../pages/business/dashboard.de').default
+  'business/setup/avv': typeof import('../pages/business/setup/avv.de').default
+  'business/setup/keys': typeof import('../pages/business/setup/keys.de').default
+  'business/setup/intro': typeof import('../pages/business/setup/intro.de').default
+  'business/setup/signup': typeof import('../pages/business/setup/signup.de').default
+  'business/setup/success': typeof import('../pages/business/setup/success.de').default
+  'business/setup/finished': typeof import('../pages/business/setup/finished.de').default
+  'business/setup/verify-key': typeof import('../pages/business/setup/verify-key.de').default
+  'business/password-reset/new': typeof import('../pages/business/password-reset/new.de').default
+  'business/password-reset/request': typeof import('../pages/business/password-reset/request.de').default
+  'business/company/[companyId]/index': typeof import('../pages/business/company/[companyId]/index.de').default
+  'business/setup/verify-key-manually': typeof import('../pages/business/setup/verify-key-manually.de').default
+  'business/company/[companyId]/checkins': typeof import('../pages/business/company/[companyId]/checkins.de').default
+  'business/company/[companyId]/area/index': typeof import('../pages/business/company/[companyId]/area/index.de').default
+  'business/company/[companyId]/area/[areaId]': typeof import('../pages/business/company/[companyId]/area/[areaId].de').default
+  'business/company/[companyId]/data-request/[dataRequestId]': typeof import('../pages/business/company/[companyId]/data-request/[dataRequestId].de').default
+}
+
+export default LocalesResources
```

Comment:
This file is automatically generated... maybe it should not be in git

Reply:
I prefer them to be in git if it improves the "out of the box" experience for developers we want to onboard.